src/pages/workspace/WorkspaceNewRoomPage.js

```diff
@@ -154,6 +155,11 @@ function WorkspaceNewRoomPage(props) {
         [translate],
     );
 
+    const roomNameInputRef = useRef(null);
+
+    // use a 600ms delay for delayed focus on the room name input field so that it works consistently on native iOS / Android
+    useDelayedInputFocus(roomNameInputRef, 600);
```

Comment:
@akinwale Is there a particular reason for 600ms timing here?

Reply:
https://github.com/Expensify/App/pull/28646#discussion_r1344589187